doc/release/0.19.0-notes.rst

```diff
@@ -2,11 +2,9 @@
 SciPy 0.19.0 Release Notes
 ==========================
 
-.. note:: Scipy 0.19.0 is not released yet!
-
 .. contents::
 
-SciPy 0.19.0 is the culmination of X months of hard work. It contains
+SciPy 0.19.0 is the culmination of 7 months of hard work. It contains
```

Comment:
Something strange going on here, it's changing the old release notes... You can do `git checkout 9749144608fb9bed49aa5 -- doc/release/0.19.0-notes.rst` and commit to revert it. Or, rebase and remove the offending commits.